app/forms/application_merits_task/involved_child_form.rb

```diff
@@ -0,0 +1,62 @@
+module ApplicationMeritsTask
```

Comment:
Might be worth moving the parent directory into the providers folder
i.e.
```
app/forms/providers/application_merits_task/involved_child_form.rb
```
Colin has done that and I was going to follow suit too.

Reply:
OK, will do.